### ISSUE TITLE
ERROR: Failed building wheel for dm-tree

### ISSUE THREAD

**Issue by Asriel31** [rlworkgroup/garage#2293]
Not for me despite I'm using anaconda3-2019.03 Python 3.7.3
I've performed conda install -c conda-forge dm-tree
Another thing that I don't understand my macosx version is 10.13.6 x86_64 High Sierra and here above the build is performed toward macosx 10.9 x86_64

Complete output (13 lines):
running bdist_wheel
running build
running build_py
creating build
creating build/lib.macosx-10.9-x86_64-3.7
creating build/lib.macosx-10.9-x86_64-3.7/tree
copying tree/init.py -> build/lib.macosx-10.9-x86_64-3.7/tree
copying tree/tree_test.py -> build/lib.macosx-10.9-x86_64-3.7/tree
copying tree/tree_benchmark.py -> build/lib.macosx-10.9-x86_64-3.7/tree
running build_ext
bazel build //tree:_tree --symlink_prefix=build/temp.macosx-10.9-x86_64-3.7/bazel- --compilation_mode=opt
unable to execute 'bazel': No such file or directory
error: command 'bazel' failed with exit status 1
ERROR: Failed building wheel for dm-tree

**Comment by krzentner** [comment]
Yeah, this breakage occurred because dm-tree's build script is broken, and the platform you're on doesn't have a pre-built dm-tree package uploaded to PyPI. Here are a few workarounds you can try:

If you don't need tensorflow support, you can install garage from [this branch](https://github.com/rlworkgroup/garage/tree/kr-tf-optional), which makes tensorflow optional.

Alternatively, the dm-tree repo has had a cmake based build for several months (but no PyPI release). You can try installing dm-tree from source, which might work.

I hope one of these options works for you.

**Comment by Asriel31** [comment]
Thanks a lot to you've been so responsive, I'm gonna try to compile the dm-tree package
